hyperspy/tests/mva/test_decomposition.py

```diff
@@ -36,37 +38,37 @@ def setUp(self):
         self.s2 = s2
         self.s12 = s12
 
-    def test_consistensy(self):
+    def test_consistency(self):
         s1 = self.s1
         s2 = self.s2
         s12 = self.s12
         s1.decomposition()
         s2.decomposition()
         s12.decomposition()
-        assert_true((s2.learning_results.loadings ==
-                     s12.learning_results.loadings).all())
-        assert_true((s2.learning_results.factors ==
-                     s12.learning_results.factors).all())
-        assert_true((s1.learning_results.loadings ==
-                     s2.learning_results.factors).all())
-        assert_true((s1.learning_results.factors ==
-                     s2.learning_results.loadings).all())
-
-    def test_consistensy_poissonian(self):
+        nose.tools.assert_true((s2.learning_results.loadings ==
+                                s12.learning_results.loadings).all())
+        nose.tools.assert_true((s2.learning_results.factors ==
+                                s12.learning_results.factors).all())
+        nose.tools.assert_true((s1.learning_results.loadings ==
+                                s2.learning_results.factors).all())
+        nose.tools.assert_true((s1.learning_results.factors ==
+                                s2.learning_results.loadings).all())
+
+    def test_consistence_poisson(self):
```

Comment:
should this not be `test_consistency_poisson` in order to be consistent with line 41 (`test_consistency`)?